server/src/main/java/org/elasticsearch/action/search/CanMatchPreFilterSearchPhase.java

```diff
@@ -386,85 +415,13 @@ protected void doRun() {
         });
     }
 
-    public void onPhaseFailure(String msg, Exception cause) {
+    private void onPhaseFailure(String msg, Exception cause) {
         listener.onFailure(new SearchPhaseExecutionException("can_match", msg, cause, ShardSearchFailure.EMPTY_ARRAY));
     }
 
-    public Transport.Connection getConnection(SendingTarget sendingTarget) {
```

Comment:
This was single use, no need to be tricky just inline it. This was a remnant of when can-match was an extension of the abstract async phase.